pkgs/games/super-tux-kart/default.nix

```diff
@@ -34,6 +34,9 @@ in stdenv.mkDerivation rec {
     curl fribidi bluez libjpeg libpng
   ];
 
+  # https://github.com/supertuxkart/stk-code/issues/3557#issuecomment-440794379
+  patches = [ ./fix-for-system-glew.patch ];
```

Comment:
This patch is taken from https://github.com/supertuxkart/stk-code/commit/3a3953f38c3555e87f3608d0291dbfccf34e9775? Would you please use `fetchPatch`? then we do not need to store the patch file ourselves.